internal/datacoord/garbage_collector.go

```diff
@@ -148,24 +147,24 @@ func (gc *garbageCollector) Resume(ctx context.Context) error {
 // work contains actual looping check logic
 func (gc *garbageCollector) work() {
 	defer gc.wg.Done()
-	ticker := time.NewTicker(gc.option.checkInterval)
-	defer ticker.Stop()
-	scanTicker := time.NewTicker(gc.option.scanInterval)
-	defer scanTicker.Stop()
+
+	// TODO: fast cancel for gc when closing.
+	// Run gc tasks in parallel.
+	gc.wg.Add(1)
+	go gc.runRecycleTaskWithPauser("meta", gc.option.checkInterval, func() {
```

Comment:
is it safe to run GC in parallel? I've no idea yet but it seems to need a double check

Reply:
it's safe to do `meta clear` and `Orphan File clear` concurrently.
We are always clear meta after clear all related file.
orphan file clearing is fenced by meta deletion.